.github/test-a-feature.md

```diff
@@ -236,3 +239,42 @@ Add description under the `@description` tag. Like:
 
 #### I want to create unit tests in separate file not through the regex
 Add your spec file into the array of files `skipSpecChecksForFiles` in `testHelper.tsx`. And put description in behavior file under `@description` tag.
+
+## Performance Tests
+
+Performance tests will measure performance, set a baseline for performance and help guard against regressions. 
+
+### Adding a Perf Test
+
+- To add a perf test, simply add a file with `.perf.` in its naming. (As of writing, `.perf.` files in `docs/src` and `packages/perf-test` are automatically consumed.)
```

Comment:
Do we want to describe the difference between the two? (docs/src perf example are visible in docsite)
But that might change soon.

Reply:
Yeah this will have to change. We'll have to update this as we go along, but for now, I wanted to document it primarily for internal use.

Reply:
We should talk about strategy here but I think ideally these tests would live alongside component implementations as similar to other types of tests. I view `packages/perf-tests` as a temporary location for comparing components.